src/main/antlr4/de/zalando/beard/BeardParser.g4

```diff
@@ -33,85 +31,80 @@ structuredStatement
     | renderStatement
     | blockStatement
     | yieldStatement
+    | extendsStatement
+    | contentForStatement
     ;
 
 yieldStatement
 locals [YieldStatement result]
-    : NL WS* LL YIELD RR
```

Comment:
I understand that the purpose of this rule was to "throw away" whitespaces before an `yield`. But only if there is a newline before, which is weird. If for example I had a file with `\s\s\s{{yield}}` than it would consider these 3 spaces as text.

So I thought it would be acceptable to just consistently consider every space outside of an "structuredStatement" as text.